Lecture 15/hw/test.py

```diff
@@ -0,0 +1,16 @@
+import requests
+
+
+def test_verify_response():
+    response = requests.get('https://api.punkapi.com/v2/beers/8')
+
+    assert response.status_code == 200
+    assert response.json()[0]['name'] == 'Fake Lager'
+    assert response.json()[0]['abv'] == 4.7
+
+
+def test_wrong_request():
+    response = requests.delete('https://api.punkapi.com/v2/beers/8')
```

Comment:
You can reuse url. For this situation with the same one better to use one variable.
For other situations something like "baseurl+your_endpoint"